usr/lib/python3/dist-packages/linuxmusterLinuxclient7/printers.py

```diff
@@ -73,16 +73,20 @@ def translateSambaToIpp(networkPath):
 # --------------------
 
 def _installPrinter(username, networkPath, name):
-    logging.debug("Installing Printer {0} on {1}".format(name, networkPath))
-    installCommand = ["lpadmin", "-p", name, "-E", "-v", networkPath, "-m", "everywhere", "-u", f"allow:{username}"]
+    logging.info("_installPrinter {0} on {1}".format(name, networkPath))
+    installCommand = ["timeout", "3", "lpadmin", "-p", name, "-E", "-v", networkPath, "-m", "everywhere", "-u", f"allow:{username}"]
```

Comment:
I think, 3 seconds may be too low in some cases of slow networks. Please change this to 15 Seconds.

Reply:
15 Sekunden halte ich für zu lang, da dann solange der login-Prozess blockiert wird, ggf, mehrfach bei mehreren Druckern. 3 Sekunden genügte bei meiner langsamen virtuellen Umgebung, ändern auf 5 Sekunden?

Reply:
I think we should at least take 10 seconds. This is just an edge case and should never happen in a properly configured environment.
But we could make it configurable once #32 is done.